tests/test_models.py

```diff
@@ -0,0 +1,12 @@
+from trailblazer.models import Config
+
+
+def test_instantiate_config():
```

Comment:
Would adding a test instantiating a config without the `analysis_host` be helpful?

Reply:
Not needed snymore